app/models/vm_or_template.rb

```diff
@@ -1884,6 +1884,10 @@ def supports_snapshots?
     false
   end
 
+  def supports_provisioning?
+    true
```

Comment:
@h-kataria We need to pick a better method name here.  The current name `supports_provisioning?` is very misleading as you are only using it to checking if the instance is connected to a provider.

I think the method you are really trying to write here is something like `has_ems?`.  @blomquisg Is there something like this already we should be using?

I think this is close to the `active?` method but that excludes templates.